### DISCUSSION TITLE
Raspberry Pi model B with 2gb discontinued!

### DISCUSSION THREAD

**Discussion by K1LBG** [WLAN-Pi/feedback#44] [Hardware - WLAN Pi R4] [answered]
Hi, trying to follow parts list at WLANPI for the R4 and cannot find a model B with 2gb at the recommended retailers.  Am I missing something?  thanks in advance

**Comment by jolla** [comment, marked as answer]
Stock is still a bit hard to come by, but I recommend checking out https://rpilocator.com
